cpp/modmesh/view/RManager.cpp

```diff
@@ -131,6 +131,16 @@ void RManager::setUpMenu()
             m_fileMenu->addAction(action);
         }
 
+        m_eulerMenu = m_mainWindow->menuBar()->addMenu(QString("Euler"));
+
+        {
+            auto app_name = QString("euler1d");
+            auto * action = new RAppAction(
+                QString("Load ") + app_name,
+                QString("Load ") + app_name,
+                QString("modmesh.app.") + app_name);
+            m_eulerMenu->addAction(action);
+        }
```

Comment:
Add `Euler` into modmesh main window toolbar.

Reply:
This is good